app/webpacker/controllers/mailing-list_controller.js

```diff
@@ -50,11 +50,11 @@ export default class extends Controller {
     return parseInt(count);
   }
 
-  dismissFeedback() {
-    return window.localStorage.setItem(feedbackDismissedKey, 'true');
+  dismissMailingList() {
+    return window.localStorage.setItem(mailingListDismissedKey, 'true');
```

Comment:
May be one for the future, but would it be better to set this on the mailing list completion page? I'm thinking if someone hits the CTA then things 'I'll do this later' and comes back to the website then they have no way of getting back to the mailing list sign up? (unless they happen across one of the CTAs in the content somewhere?)

Reply:
Good point. Perhaps a ever-present link in the footer might make sense here?